CONTRIBUTING.md

```diff
@@ -3,6 +3,23 @@ Contributing
 
 Thanks for contributing!
 
+## Before you contribute
+
+This package is a fork of [fast-deep-equal](https://github.com/epoberezkin/fast-deep-equal). This library has added handling for React. Before contributing, _please make sure the issue relates directly to this library and not fast-deep-equals_.
+
+We encourage pull requests concerning:
+
+* React features not handled in this library
```

Comment:
Let's add another bullet for the thing you just did: `* Integrating updates from fast-deep-equal`